cometd-java/cometd-java-server/cometd-java-server-common/src/main/java/org/cometd/server/BayeuxServerImpl.java

```diff
@@ -1300,8 +1302,20 @@ public void setDetailedDump(boolean detailedDump) {
         _detailedDump = detailedDump;
     }
 
+    @ManagedAttribute("The number of parallel threads used to perform sweeping")
+    public int getSweepThreads()
+    {
+        return _sweepThreads;
+    }
+
+    public void setSweepThreads(int sweepThreads)
+    {
+        _sweepThreads = sweepThreads;
+    }
```

Comment:
Please document the new parameter in the documentation, in the server parameters table.

Reply:
Done.